### ISSUE TITLE
clGetContextInfo's last parameter is "size_t" when it should be "size_t*"

### ISSUE THREAD

**Issue by HeavyMetalCookies** [KhronosGroup/OpenCL-Docs#64]
The function prototype/signature is wrong. Found while writing bindings and comparing
documentation against khronos' cl.h.

The last parameter listed in clGetContextInfo should be a pointer.
https://www.khronos.org/registry/OpenCL/sdk/1.0/docs/man/xhtml/clGetContextInfo.html

cl_int clGetContextInfo (	cl_context context,
 	cl_context_info param_name,
 	size_t param_value_size,
 	void *param_value,
 	size_t param_value_size_ret)

**Comment by bashbaug** [comment]
I've checked the specs and they all appear to be correct, but this does appear to be a typo in the online man pages for OpenCL 1.0 only. @oddhack, are you the right person to get this updated?

Side note: If you're working on OpenCL bindings you may be interested in an updated machine readable grammar for the OpenCL APIs.  It's not quite committed to master yet, but it's fairly complete.  You may be able to use it to auto-generate some (if not all) of your bindings:

https://github.com/KhronosGroup/OpenCL-Docs/blob/refpages/xml/cl.xml

Thanks for reporting this issue!

**Comment by HeavyMetalCookies** [comment]
Yes. Typo in 1.0 only. I am a novice programmer. I don't know anything
about machine readable grammar, but I get the idea from the context.
I am righting bindings for each function as I need it. And the manual work
is good practice for me I think.

Thanks for the reply!
